canvas_modules/common-canvas/locales/command-actions/locales/eo.json

```diff
@@ -16,14 +16,14 @@
 	"action.createNodeOnLink": "[Esperanto~Create {node_label} node and insert into link~~~~~~~eo]",
```

Comment:
You haven't made a corresponding change in this file for the color comments message.